src/Invoice/Inv/Service/PlaywrightDocumentHtmlBuilder.php

```
@@ -0,0 +1,56 @@
+     * correct for mPDF, which reads image files straight off disk, but a
+     * real browser can't load a bare filesystem path as an image URL.
+     * '@public' is the webroot itself, so stripping it leaves the correct
+     * web-relative path.
+     */
+    private function rewriteLogoSrcForBrowser(string $html): string
+    {
+        $publicPath = $this->aliases->get('@public');
+        $pattern = '#src="' . preg_quote($publicPath, '#') . '([^"]*)"#i';
+        return preg_replace_callback(
+            $pattern,
+            static fn (array $m): string => 'src="/' . ltrim(str_replace('\\', '/', $m[1]), '/') . '"',
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $m. Configured minimum length is 3. (undefined)

(ShortVariable)